Add Introduction Page (#32)
LibertyDSNP/frequency-docs: pages/index.md

## Work in Progress

# Introduction

> Frequency is the first implementation of [The Decentralized Social Network Protocol (DSNP)](https://www.dsnp.org/introducing-dsnp.html) operating as a Polkadot parachain. 

Reviewer: ```suggestion
Frequency is the first implementation of [The Decentralized Social Network Protocol (DSNP)](https://www.dsnp.org/) operating as a Polkadot parachain. 
```

Frequency delivers essential tooling to create a next generation of people-centric social networking apps that offer consumers more control and choice. 
In other words, it enables users to interact in a public digital space while retaining agency over their data. 
It provides scalable message passing as a backbone of new shared protocols that create value through cooperation and user choice. 

Here you can find detailed documentation for Frequency, official implementation specifications, associated specifications, as well as a roadmap and proposals for future extensions.

## Core Ideas

Frequency aims to return agency to the user amongst an ecosystem of rampant misinformation, censorship from state and corporate actors, and near monopolies from select players with absolute power over social engagements. 
Frequency, as an implementation of DSNP, strives to solve these challenges by reshaping online social interactions through prioritizing: 

1. **Different Classes of Transactions**
	* Data Focused Transactions
 
2. **Broadcast as a Key to Decentralization**
	* Announcements and Metadata
	
3. **Different Economic Structure**

> 	* Capacity

Author: Can't link to Capacity page until it is committed to Summary page on the Intro branch

	
4. **Collaborative Standardization**
	* Shared Schemas
 

## The Problem: 

Social networking has wholly transformed digital communication for the modern age. 
The industry has become dominated by a handful of global private companies, growing rich off the data of their users. 
These data repositories sparked significant user insights which soon grew into a web of manipulation, exploitation, and polarization. 
The evolution of these secretive agencies made clear that the social infrastructure needed to change and return agency back to the users. 

Yet when blockchain technology is applied to high-volume, low-value transactions associated with a social network, the cost quickly overwhelms business models. 

## The Solution:

The solution necessitates developing a new foundation and infrastructure to secure individual’s data rights and opens innovation beyond the current select few. 

> The project originated with [Unfinished](https://www.unfinishedlabs.io/), a social impact organization with a network of partners engaged in technology, academia, social impact, and the arts. The first phase of work focused on delivering the social network as core internet functionality.  The Decentralized Social Network, DSNP serves as a public resource entirely independent of financial incentives. Frequency is the second strange of work and unlocks decentralized social networking at scale for the first time.

Reviewer: ```suggestion
The project originated with [Unfinished](https://www.unfinished.com/), a social impact organization with a network of partners engaged in technology, academia, social impact, and the arts. The first phase of work focused on delivering the social network as core internet functionality.  The Decentralized Social Network, DSNP serves as a public resource entirely independent of financial incentives. Frequency is the second strange of work and unlocks decentralized social networking at scale for the first time.
```

Frequency builds on blockchain technology to enable users to interact in a public digital space and retain agency over their data. 
A public social graph disconnects financial incentivization completely, establishing an essential foundation for the future of social networks. 
Frequency changes the way blockchain transactions are priced to leverage a network scale suitable for social networking. 
This approach allows technology to be cost-efficiently used for a new range of high-frequency, massive-scale decentralized apps and services.

> To learn more about decentralized social networks visit [DSNP.org](https://www.dsnp.org/introducing-dsnp.html)

Reviewer: ```suggestion
To learn more about decentralized social networks visit [DSNP.org](https://www.dsnp.org/)
```

> ## Scalability:

Author: Could see this being moved to a different page for additional information to be contributed. Could overlap with the future roadmap content.

Author: Perhaps under Learn -> Vision section

> Building Frequency as a [Polkadot](https://wiki.polkadot.network/docs/getting-started) parachain offers a solution to the challenge of scaling a social networking platform. 

Reviewer: ```suggestion
Building Frequency as a [Polkadot](https://wiki.polkadot.network/) parachain offers a solution to the challenge of scaling a social networking platform. 
```

Polkadot's fixed price consensus combined with the relay chain and parachain infrastructures offer capability to manage costs more effectively than other blockchain ecosystems. 

Frequency addresses scalability primarily through batching and delegation. 
These schemas significantly reduce the number of transactions the main blockchain needs to verify and record, which in turn speeds up the processing time of the remaining transactions.

> ## Financial Incentive Freedom for Applications on DSNP and Frequency: 

Author: Same as Scalability - Depends on how much content we want to have on the introduction page - I feel like this could be broken up into smaller pieces.

> Removal of financial incentives is a key pillar of DSNP and Frequency’s implementation framework. 
> 
> Implementation elements for this concept are currently under development.

Reviewer: Not sure how to word this section. Perhaps just coming soon and then we write the summary after we have the other content.
```suggestion
Coming soon
```

## Work in Progress

# Introduction

Frequency is the first implementation of The Decentralized Social Network Protocol (DSNP) operating as a Polkadot parachain. 
Frequency enables users to interact in a public digital space while retaining agency over their data. 

> It provides scalable message passion as a backbone of new shared protocols that create value through cooperation and user choice. 

Reviewer: We might want to re-write the intro based on the content on https://www.frequency.xyz/

```suggestion
It provides scalable message passing as a backbone of new shared protocols that create value through cooperation and user choice. 
```

Author: Do we want to rewrite to go more into the "why" behind DSNP and Frequency or more to echo the writing style and content from this?

Reviewer: @ebroady I think we can add some. Let's keep to the writing style of the docs, so less "marketing" tone, and align it with the sections of the docs.

Here you can find detailed documentation for Frequency, official implementation specifications, associated specifications, as well as a roadmap and proposals for future extensions.

## Core Ideas

Frequency aims to return agency to the user in a modern ecosystem consisting of rampant misinformation, censorship from state and corporate actors, and near monopolies from select players with absolute power over social engagements. 
Frequency, as an implementation of DSNP, strives to solve these challenges in reshaping online social interactions.

1. **Different Classes of Transactions**
	* Data Focused Transactions
 
2. **Broadcast as a Key to Decentralization**
	* Announcements and Metadata
	
3. **Different Economic Structure**
	* Capacity
	
4. **Collaborative Standardization**

> 	* Smart Schemas

Reviewer: Trying out a different word over "smart". Thoughts?
```suggestion
	* Shared Schemas
```

A public social graph disconnects financial incentivization completely, establishing an essential foundation for the future of social networks. 
We change the way blockchain transactions are priced to leverage a network scale suitable for social networking. 
This approach makes decentralized social networks production-ready for the first time. 

To learn more about decentralized social networks visit [DSNP.org](https://www.dsnp.org/introducing-dsnp.html)

## Scalability:

Building Frequency as a [Polkadot](https://wiki.polkadot.network/docs/getting-started) parachain offers an elegant solution to the challenge of scaling a social networking platform. 
Polkadot’s fixed price consensus combined with the relay chain and parachain infrastructures offer capability to manage costs more effectively than other blockchain ecosystems. 

Frequency addresses scalability primarily through batching and delegation. 
These schemas significantly reduce the number of transactions the main blockchain needs to verify and record, which in turn speeds up the processing time of the remaining transactions.

> ## Lack of Financial Incentive for dApps: 

Reviewer: Not sure about this suggestion, but I think we need to re-write this section. Let's talk about it.

```suggestion
## Financial Incentive Freedom for Applications on DSNP and Frequency: 
```
